pkg/manager/member/dm_worker_failover_test.go

```diff
@@ -119,33 +128,34 @@ func TestWorkerFailoverFailover(t *testing.T) {
 			expectFn: func(t *testing.T, dc *v1alpha1.DMCluster) {
 				g := NewGomegaWithT(t)
 				g.Expect(len(dc.Status.Worker.FailureMembers)).To(Equal(1))
+				g.Expect(dc.Status.Worker.FailoverUID).To(BeEmpty())
```

Comment:
Why there is a failure member but the uid is empty?

Reply:
The UID be generated together with append FailureStores, `dm-worker-1` already exist there

Reply:
In this case, if failover occurred before upgrading to the TiDB Operator including this fix, the user still cannot set the recoverByUID, right? What about we also generate the uid for this case, e.g. `if worker.Stage == v1alpha1.DMWorkerStateOffline && time.Now().After(deadline), check and set the status`.